politeiawww/api/www/v1/v1.go

```diff
@@ -483,8 +483,9 @@ type Version struct{}
 // this backend supports and additionally the route to the API and the public
 // signing key of the server.
 type VersionReply struct {
-	Version           uint   `json:"version"`           // lowest supported WWW API version
-	Route             string `json:"route"`             // prefix to API calls
+	Version           uint   `json:"version"`           // Lowest supported WWW API version
+	Route             string `json:"route"`             // Prefix to API calls
+	GithubVersion     string `json:"githubversion"`     // Github version in which Pi was built
```

Comment:
Why is this named GithubVersion? The politeia build information shouldn't have anything to do with Github?  Github is just where we host the repo.

Reply:
Right, thought of mentioned from where the build version was coming from in the naming, but indeed doesn't suit well. Changed this to BuildVersion, which is were we get this info.

Reply:
Much better.